### PR TITLE
FW continuity fix

### PR DESCRIPTION
**Pull request by jafdy**
could you check before merge

### CODE REVIEW

**Comment by jafdy** [comment]
Done. Thanks.

File: data/free worlds intro.txt [hunk 1]
```diff
@@ -808,6 +808,12 @@ mission "FW Katya 3"
 					goto no
 			
 			label yes
+			`	"Are you being sarcastic?" she asks.
+			choice
+				`	"Yes"
+					goto no
+				`	"No"
```
Comment (toilethinges) [inline data/free worlds intro.txt]: For this and the above, it looks like there are no end ` marks, and they need a period before the end quotation.

**Comment by ItsNickBarry** [comment]
It should be presumed that Katya is astute enough to detect sarcasm without the need to ask.  The sarcastic comment suggests a disinterest in the technical aspects of obscure parliamentary resolutions, and Katya accordingly responds with less detail.

**Comment by jafdy** [comment]
@toilethinges Thanks. It shows this is my first time.

@ItsNickBarry That is the main thing I've thought about in this pull. I can change it but it depends on what everyone (or endless) thinks.

**Comment by jafdy** [comment]
I have broken GitHub. I now don't have write access to my own pull. I have some ideas so ill start a new pull and try not to brake that one.

**Pull request closed without merging**